README.md

```diff
@@ -15,225 +13,24 @@ OpenAPI 3 CLI toolbox with rich validation and bundling features.
-
-    parameter-description: off
-    no-path-trailing-slash: error
-    path-declaration-must-exist: error
-    path-not-include-query: error
-    path-parameters-defined: error
-    operation-description: off
-    operation-2xx-response: warn
-    operation-operationId-unique: error
-    operation-parameters-unique: error
-    operation-tag-defined: off
-    operation-security-defined: warn
-    operationId-valid-in-url: error
-    operation-singular-tag: off
-
-    no-example-value-and-externalValue: error
-
-    no-unused-components: warn
-    no-unresolved-refs: error
-    no-enum-type-mismatch: error
-
-    boolean-parameter-prefixes: off
-    paths-kebab-case: off
-    spec: 'error'
-```
-
-All of the rules are configurable in terms of disabling or changing their severity, or even defining pinpoint exclusions.
-
-Here is an example of a modified use `.redocly.yaml` file:
-
-```yaml
-lint:
-  rules:
-    parameter-description: warn
-    operationId-valid-in-url: off
-```
-
-Each rule can be turned `off`. In addition, you can control the log-level severity, between `warn` and `error`.
-
-Enabling a rule:
-```yaml
-lint:
-  rules:
-    <rule-name>: error
-```
-
-Disabling a rule:
-```yaml
-lint:
-  rules:
-    <rule-name>: off
-```
-
-#### Rules Severity Levels
-
-Changing the severity of a rule when rule accepts options:
-
-```yaml
-lint:
-  rules:
-    <rule-name>:
-      level: <value>
-      # ...rule options
-```
-
-or
-
-```yaml
-lint:
-  rules:
-    <rule-name>: <value>
-```
-
-Possible values are:
-
-* warn
-* error
-
-#### Ignore file
-
-This file instructs Redocly's linter to ignore the rules contained for specific parts of your API.
-
-You can exclude some known errors to not show up/break you validation using ignore file.
-Ignore file should be named `.redocly.lint-ignore.yaml` and should be placed next to `.redocly.yaml`.
-
-```yaml
-"filename.yaml":
-  rule-id-1:
-    - "#/excluded/json/pointer"
-    - "#/excluded/json/pointer"
-  rule-id-2:
-    - "#/excluded/json/pointer"
-    - "#/excluded/json/pointer"
-```
-
-Ignore file can be autogenerated to add all problems to it by running `openapi lint --generate-ignore-file`
-
-#### Built-in Rules
+## What makes this tool different
 
-[Read the docs](docs/rules.md) for the built-in rules.
+Unlike other OpenAPI linters, `@redocly/openapi-cli` defines the possible type tree of a valid OpenAPI definition and then traverses it. This approach is very similar to how compilers work and results in major performance benefits over other approaches. Extend functionality at different points in the lifecycle with preprocessors, rules, and decorators by following the [visitor pattern](https://en.wikipedia.org/wiki/Visitor_pattern).
 
-### Advanced
 
-[Plugins](docs/plugins.md)
-[Custom rules](docs/custom-rules.md)
-[Decorators](docs/decorators.md)
-[Preprocessors](docs/preprocessors.md)
-[Type extensions](docs/type-extensions.md)
+## [Read the docs](https://redoc.ly/docs/cli/)
```

Comment:
I would also add TLDR usage section, basically

`npx @redocly/openapi-cli lint path-to-file.yaml`